### PR TITLE
Include workflow for ui lib unittests

### PR DESCRIPTION
**Pull request by kimwnasptd**
As part of setting up our CI/CD #5482 we want to extend the prow config to also run the unittests of the shared ui as a presubmit job.

/cc @PatrickXYS

### CODE REVIEW

**Review by PatrickXYS — APPROVED** [review]
/lgtm
/approve

**Comment by PatrickXYS** [comment]
@kimwnasptd How about adding us into root folder's OWNERS to grant permission.

Otherwise, every time you change prow_config.yaml, you'll need to ask root OWNERS' permission

**Comment by kimwnasptd** [comment]
@PatrickXYS you have a point. I'll make a PR to add us in the rook OWNERS file, so that we can move continue our CI/CD effort

**Comment by kimwnasptd** [comment]
@PatrickXYS I added another commit here that removes the GCP workflow. The tests no seem to pass, although I can't find any relative logs from the underlying workflow.

Is it possible to see the logs from each Argo workflow task under the `Build Log` section in Prow's site? [ after clicking `Details` on the right of the bot ]

**Comment by kimwnasptd** [comment]
@PatrickXYS I think we are set for this PR but feel free to add any more comments/thoughts you might have.

@Bobgy this is the PR that fixes the broken tests in `kubeflow/kubeflow`. As mentioned above we
* removed the older GCP workflow
* added another one that tests some common frontend code we have

So once we merge this feel free to also move forward with your work and remove `kubeflow/kubeflow` from the older infra https://github.com/kubeflow/kubeflow/issues/5482#issuecomment-782573046. So could you approve the PR in order to move forward?

File: prow_config.yaml [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-    name: unittests
-    params:
-      workflowName: unittest
+  # Run unittests for the shared frontend code
+  - py_func: kubeflow.kubeflow.ci.shared_ui_tests.create_workflow
+    name: shared_ui_tests
+    job_types:
+      - presubmit
+    include_dirs:
+      - components/crud-web-apps/common/frontend/kubeflow-common-lib/*
```
Comment (PatrickXYS) [inline prow_config.yaml:15]: The `include_dirs` will contain the directory that you wish to kick off the workflow.

But apprently in this PR, you haven't made any change to `components/crud-web-apps/common/frontend/kubeflow-common-lib/*`

Then there's no workflow

**Comment by PatrickXYS** [comment]
> Is it possible to see the logs from each Argo workflow task under the Build Log section in Prow's site?

In short, it's by design, because those Argo workflow will flood the Build Log to say 10,000+ lines of logs, you don't want to check so.

The reason you haven't see any logs in Argo Workflow is because this PR doesn't kick off any workflow.

**Review by Bobgy — APPROVED** [review]
/lgtm
/approve

**Pull request merged**